src/api/.rubocop.yml

```diff
@@ -81,3 +81,239 @@ Rails/BulkChangeTable:
     - 'db/migrate/20170912140257_change_kiwi_packages_columns_from_big_int_to_int.rb'
     - 'db/migrate/20171107125828_rename_kiwi_preference_types_to_kiwi_preferences.rb'
     - 'db/migrate/20180516074538_explicitly_set_charset.rb'
+
+Style/MethodCallWithArgsParentheses:
+  Enabled: true
+  Exclude:
+    - 'app/views/**/*'
+    - 'db/migrate/**/*'
+  # the cop doesn't come with a well defined list of good methods, so this
```

Comment:
what about trying to short this list here by adding it upstream, this is not really OBS related as everybody enabling this cop will have a similar list. Specially for rspec I think it would make sense: https://github.com/rubocop-hq/rubocop-rspec/issues/667